compiler/fir/java/src/org/jetbrains/kotlin/fir/java/deserialization/JvmBinaryAnnotationDeserializer.kt

```diff
@@ -33,6 +35,7 @@ class JvmBinaryAnnotationDeserializer(
     private data class MemberAnnotations(val memberAnnotations: Map<MemberSignature, MutableList<FirAnnotationCall>>)
 
     private enum class CallableKind {
+        FUNCTION,
```

Comment:
Seems only GETTER/SETTER are relevant kinds. Maybe use something like OTHER or NON_PROPERTY_ACCESSOR, instead of FUNCTION?

Reply:
Yeah, this is not quite relevant. Thanks for the suggestion, I'll call it `OTHERS`.